src/locale/cn/help.json

```diff
@@ -0,0 +1,349 @@
+{
+  "help": {
+    "overview": {
```

Comment:
I think omit these keys (`overview`, `payouts`, etc) and just have a big list of help items.

Reply:
that make sense